apps/google-analytics-4/frontend/src/components/config-screen/HomeAnalyticsPage.tsx

```diff
@@ -0,0 +1,218 @@
+import { useCallback, useState, useEffect } from 'react';
+import { AppExtensionSDK } from '@contentful/app-sdk';
+import { Box } from '@contentful/f36-components';
+import { css } from 'emotion';
+import omitBy from 'lodash/omitBy';
+import { useSDK } from '@contentful/react-apps-toolkit';
+import tokens from '@contentful/f36-tokens';
+import ApiAccessPage from './api-access/ApiAccessPage';
+import {
+  convertServiceAccountKeyToServiceAccountKeyId,
+  convertKeyFileToServiceAccountKey,
+  AssertionError,
+} from '../../utils/serviceAccountKey';
+import type { AppInstallationParameters, ServiceAccountKey, ServiceAccountKeyId } from '../../types';
+import AboutSection from './header/AboutSection';
+import ConfigurationPage from './configuration/ConfigurationPage';
+import Splitter from './utils/Splitter';
+import GoogleAnalyticsIcon, { googleAnalyticsBrand } from './utils/GoogleAnalyticsIcon';
+
+const styles = {
+  body: css({
+    height: 'auto',
+    minHeight: '65vh',
+    margin: '0 auto',
+    marginTop: tokens.spacingXl,
+    padding: `${tokens.spacingXl} ${tokens.spacing2Xl}`,
+    maxWidth: tokens.contentWidthText,
+    backgroundColor: tokens.colorWhite,
+    zIndex: 2,
+    boxShadow: '0px 0px 20px rgba(0, 0, 0, 0.1)',
+    borderRadius: '2px',
+  }),
+  background: css({
+    display: 'block',
+    position: 'absolute',
+    zIndex: -1,
+    top: 0,
+    width: '100%',
+    height: '300px',
+    backgroundColor: googleAnalyticsBrand.primaryColor,
+  }),
+};
+
+const HomeAnalyticsPage = () => {
+  const [parameters, setParameters] = useState<AppInstallationParameters>({
```

Comment:
Common Pattern Comment: General component structure
1. Desctructure your props (if there are any)
2. Usage of hooks (useState, useCallback, useEffect) - with useState generally being first up top
3. Function definitions in the component
4. JSX